src/components/TagSearch.vue

```diff
@@ -1,17 +1,26 @@
 <template>
-  <article class="button-wrap">
-    <button type="button" name=""
-            v-for="item in tagList" :key="item"
-            @click="tagSearching(item)">
-      {{item}}
-    </button>
-  </article>
+  <section>
+    <article class="button-wrap" v-for="(item, key) in tagList" :key="key">
+      <button type="button" name="" :class="item.open ? 'open' : ''"
+        @click="openList(key)">
+        {{`${key}: ${item.select}`}}
```

Comment:
```suggestion
        {{`${key}: ${item.select}`}}
```
이거는 추후에 UI에 따라 함수화 시켜도 되겟네요

Reply:
이건 추후에 봐야겠네요